tests/e2e/fullsync_test_for_block_volume.go

```diff
@@ -803,22 +809,29 @@ var _ bool = ginkgo.Describe("full-sync-test", func() {
 			}
 		}()
 
-		if guestCluster {
-			volHandle = getVolumeIDFromSupervisorCluster(volHandle)
-			gomega.Expect(volHandle).NotTo(gomega.BeEmpty())
-		}
-
 		ginkgo.By("create a pod pod1, using pvc1")
 		pod, _ := createPODandVerifyVolumeMount(f, client, namespace, pvc, volHandle)
 		defer func() {
 			err := fpod.DeletePodWithWait(client, pod)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
 
+		pod, err = client.CoreV1().Pods(namespace).Get(ctx, pod.Name, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Why are you getting the pod details again, pod reference is already available in line number 813

Reply:
Yes removed it.